libs/i18n/src/lib/languages/polish.ts

```diff
@@ -5,537 +5,543 @@ import { FdLanguage } from '../models/lang';
  */
 export const FD_LANGUAGE_POLISH: FdLanguage = {
     coreCarousel: {
-        leftNavigationBtnLabel: 'Go to previous item',
-        rightNavigationBtnLabel: 'Go to next item'
+        leftNavigationBtnLabel: 'Idź do poprzedniego elementu',
+        rightNavigationBtnLabel: 'Idź do nestępnego elementu'
     },
     coreDatePicker: {
-        dateInputLabel: 'Date input',
-        dateRangeInputLabel: 'Date range input',
-        displayCalendarToggleLabel: 'Open picker',
-        valueStateSuccessMessage: 'Value state Success',
-        valueStateInformationMessage: 'Value state Information',
-        valueStateWarningMessage: 'Value state Warning',
-        valueStateErrorMessage: 'Value state Error'
+        dateInputLabel: 'Data',
+        dateRangeInputLabel: 'Przedział czasowy',
+        displayCalendarToggleLabel: 'Otwórz selektor',
+        valueStateSuccessMessage: 'Value state Success', // I have no idea what it is about (?)
```

Comment:
It is for a hidden div indicating to the screenreader that the datepicker input has a success state

![Screen Shot 2022-09-02 at 1 42 19 PM](https://user-images.githubusercontent.com/2471874/188225670-579c5b9e-44f5-4fcb-aa58-d9df28cc2b4b.png)

Reply:
sure but even in english I never have I ever hit into the labels like:

```
        valueStateSuccessMessage: 'Value state Success',   
        valueStateInformationMessage: 'Value state Information', 
        valueStateWarningMessage: 'Value state Warning',  
        valueStateErrorMessage: 'Value state Error' 
```

When the `Value state Success` makes some sense, the `Value state InformationWarrning` even in English for me is odd, so if I had to translate I would be:

```
        valueStateSuccessMessage: 'Stan wartości Poprawna',   
        valueStateInformationMessage: 'Stan wartości Informacja', 
        valueStateWarningMessage: 'Stan wartości Ostrzeżenie',  
        valueStateErrorMessage: 'Stan wartości Niepoprawna' 
```